.travis/coverity/travis-tasks.sh

```diff
@@ -4,6 +4,9 @@
 set -e
 set -x
 
+if [ -e /etc/centos-release ]; then
+    COMMON_MESON_ARGS="-Dtest_dirty_git=false -Ddeveloper_build=false -Dpython_name=python3.6"
+fi
```

Comment:
Should I also update `.travis/coverity/Dockerfile.tmpl` to use `yum` instead of `dnf` for backwards compatibility with CentOS 7? Or get even trickier with some `m4` macro ugliness to use `yum` _only_ for CentOS 7?

Reply:
Just use `yum` for both on CentOS. It'll be easier and I see no reason to believe that the name will change in the future; they're both just aliases in RHEL 8.

Reply:
Done and done in my latest push.